.github/workflows/pull-request-workflow.yml

```diff
@@ -2,7 +2,7 @@ name: Pull Request Workflow
 on: [pull_request]
 jobs:
   run-tests:
-    runs-on: macos-11
+    runs-on: macOS-latest
```

Comment:
Needs to fetch all the tags as we are now using `LIB_VERSION`.
Docs: https://github.com/actions/checkout#usage